model/utils_test.go

```diff
@@ -81,3 +81,27 @@ func TestEtag(t *testing.T) {
 		t.Fatal()
 	}
 }
+
+var hashtags map[string]string = map[string]string{
+	"#test":         "#test",
+	"test":          "",
+	"#test123":      "#test123",
+	"#123test123":   "",
+	"#test-test":    "#test-test",
+	"#test?":        "#test",
+	"hi #there":     "#there",
+	"#bug #idea":    "#bug #idea",
+	"#bug or #gif!": "#bug #gif",
+	"#hüllo":        "#hüllo",
+	"#?test":        "",
+	"#-test":        "",
+	"#yo_yo":        "#yo_yo",
```

Comment:
Can you add a couple more tests here? `#v1.3-rc2` should return `#v1.3-rc2` and `#bug. #gif.` should return `#bug #gif`.